src/cmd/linuxkit/pkg_build.go

```diff
@@ -44,7 +45,8 @@ func pkgBuildPush(args []string, withPush bool) {
 	builders := flags.String("builders", "", "Which builders to use for which platforms, e.g. linux/arm64=docker-context-arm64, overrides defaults and environment variables, see https://github.com/linuxkit/linuxkit/blob/master/docs/packages.md#Providing-native-builder-nodes")
 	builderImage := flags.String("builder-image", defaultBuilderImage, "buildkit builder container image to use")
 	builderRestart := flags.Bool("builder-restart", false, "force restarting builder, even if container with correct name and image exists")
-	buildCacheDir := flags.String("cache", defaultLinuxkitCache(), "Directory for storing built image, incompatible with --docker")
+	cacheDir := flagOverEnvVarOverDefaultString{def: defaultLinuxkitCache(), envVar: envVarCacheDir}
+	flags.Var(&cacheDir, "cache", fmt.Sprintf("Directory for caching and finding cached image, overrides env var %s", envVarCacheDir))
```

Comment:
Probably it would be nice to show default as well in usage string

Reply:
flags should do that automatically. But it doesn't, apparently. Ignoring for now, until we can refactor all of the flags out.